modules/services/pulseeffects.nix

```
@@ -0,0 +1,54 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+
+  cfg = config.services.pulseeffects;
+
+  presetOpts = optionalString (cfg.preset != "") "--load-preset ${cfg.preset}";
+
+in {
+  meta.maintainers = [ maintainers.jonringer ];
+
+  options.services.pulseeffects = {
+    enable =
+      mkEnableOption "Pulseeffects daemon. Used for advanced audio mixing.";
```

Need to remove the final `.` since `mkEnableOption` adds it automatically.

Might also be worth rephrasing to something like
```suggestion
      mkEnableOption "Pulseeffects daemon; for advanced audio mixing";
```

---

I just removed the extraneous wording.